samples/Exporters/Web/OpenTelemetry.Exporter.ApplicationInsights.Web.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc5" />

```

Comment:
is it relevant for the demo purposes?

Reply:
For my goals - yes. It's a demonstration that will be tracked not only api calls, but static files as well